.eslintignore

```diff
@@ -5,3 +5,10 @@ dist/
 nuxt-template-overrides/
 test/tapes
 coverage/
+node_modules
+.nuxt
+.nuxt-storybook
+!.github
+!.husky
+storybook-static
+.idea
```

Comment:
Doesn't ESLint automatically ignore the Git-ingored files?

Reply:
It's meant to but I suspect something is broken in whatever current version of ESLint we have is because locally for me it was not ignoring them :disappointed: